src/GeneralStateTestsFiller/stCreateTest/CREATE_HighNonceDelegatecallMinus1Filler.yml

```diff
@@ -0,0 +1,156 @@
+# The test calls CREATE from an account with max allowed nonce - 1.
```

Comment:
this you can put into
```
 _info:     
    comment: ""
```

sectino this way it will pring in generated test for the devs to read